test/Microsoft.OpenApi.Tests/Services/OpenApiFilterServiceTests.cs

```diff
@@ -19,27 +19,37 @@ public OpenApiFilterServiceTests()
         }
 
         [Theory]
-        [InlineData("users.user.ListUser")]
-        [InlineData("users.user.GetUser")]
-        [InlineData("administrativeUnits.restore")]
-        [InlineData("graphService.GetGraphService")]
-        public void ReturnFilteredOpenApiDocumentBasedOnOperationIds(string operationId)
+        [InlineData("users.user.ListUser", null)]
+        [InlineData("users.user.GetUser", null)]
+        [InlineData("administrativeUnits.restore", null)]
+        [InlineData("graphService.GetGraphService", null)]
+        [InlineData(null, "users.user")]
+        [InlineData(null, "applications.application")]
+        [InlineData(null, "reports.Functions")]
+        public void ReturnFilteredOpenApiDocumentBasedOnOperationIds(string operationIds, string tags)
```

Comment:
To fully describe the test:

```suggestion
        public void ReturnFilteredOpenApiDocumentBasedOnOperationIdsAndTags(string operationIds, string tags)
```

Reply:
You can also add tests for:

`[InlineData(null, "*")]`
`[InlineData("*", null)]`

Reply:
Yeah sure.